scripts/push-prod.sh

```diff
@@ -1,8 +1,8 @@
 #!/bin/bash
 set -e
 
+# Login with `oc login ...` before running.
+oc project officehours
 docker login -u `oc whoami` -p `oc whoami -t` docker-registry.webplatformsunpublished.umich.edu
```

Comment:
Nit-pick: `docker-registry.webplatformsunpublished.umich.edu` could be made into a variable, but it's not critical given the limited purpose of these scripts.